### PR TITLE
Fix missing stat desc

### PR DESCRIPTION
**Pull request by haslinghuis**
Adding name, text and desc which were omitted in #3615

### CODE REVIEW

**Review by nerdCopter — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by chmelevskij — APPROVED** [review]

**Comment by HThuren** [comment]
@haslinghuis 
Thank you for this PR, I have spent hours to understand OSD data structure, IO was in fact on track regarding missing names, and then I saw your change, so great.

**Pull request merged**
